### ISSUE TITLE
datasets not working with angular 15 and echarts 

### ISSUE THREAD

**Issue by swapnil0545** [xieziyu/ngx-echarts#388]
Hi,

I am trying to use datasets to show up charts as shown in this example.
https://echarts.apache.org/handbook/en/concepts/dataset#define-data-in-dataset

But it is not working for me.
Any help will be really appreciated.

Thanks.

**Comment by xieziyu** [comment]
Hi @swapnil0545 

I add an example that shows how to use datasets in the [demo page](https://xieziyu.github.io/ngx-echarts/#/basic/basic-usage). It also shows how to update the dataset using `[merge]`

**Comment by swapnil0545** [comment]
Its still the same. I was having 15.0.1 and now I updated to 15.0.2. I am looking into reproducing it now. I

**Comment by swapnil0545** [comment]
Thank @xieziyu for faster response. I started looking at the source code of the demo page why it was working there and not on mines. I was using SVG renderer and not all the components were imported.
This issue happens because usually when I miss importing barchart... it gives me error that barchart import is missing.
But for dataset it doesnt throw any error.

When I imported the datasetcomponent it started working fine as intended.

import {
  TitleComponent,
  TooltipComponent,
  GridComponent,
  LegendComponent,
  ToolboxComponent,
  **DatasetComponent**
} from 'echarts/components';
// Import the Canvas renderer, note that introducing the CanvasRenderer or SVGRenderer is a required step
import { SVGRenderer } from 'echarts/renderers';
import Marcaron from './marcaron';
import { FilterComponent } from './filter/filter.component';
import 'echarts/lib/component/dataZoom';
import { NgSelectModule } from '@ng-select/ng-select';
echarts.use([
  TitleComponent,
  TooltipComponent,
  GridComponent,
  LineChart,
  SVGRenderer,
  LegendComponent,
  BarChart,
  PieChart,
  ToolboxComponent,
  **DatasetComponent**
]);
